bots/images/scripts/ubuntu-stable.install

```diff
@@ -2,6 +2,13 @@
 
 set -e
 
+# HACK: TODO file some issue for this
```

Comment:
We already have the downstream issue: https://launchpad.net/bugs/1826187 . I followed up there to point out that workaround/root cause.